flask/app/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.7-alpine 
+WORKDIR /app 
+COPY . /app 
+RUN pip3 install -r requirements.txt 
```

Comment:
Can you start with the installation of requirements before copying the source code to take advantage of layers caching

Reply:
Done